tools/plink_process_binary.cwl

```diff
@@ -0,0 +1,129 @@
+class: CommandLineTool
+cwlVersion: v1.2
+id: plink_process_binary
+doc: |-
+  Process a plink binary file
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/plink:1.90b7.2'
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+baseCommand: []
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      plink
+  - position: 3
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.genome != null ? "--genome" : "")
+  - position: 5
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.check_sex != null ? "--check-sex" : "")
+  - position: 7
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.mendel != null ? "--mendel": "")
+inputs:
+  input_bed: { type: 'File', inputBinding: { position: 2, prefix: "--bed" }, doc: "Binary BED file for plink." }
+  input_bim: { type: 'File', inputBinding: { position: 2, prefix: "--bim" }, doc: "Binary BIM file for plink." }
+  input_fam: { type: 'File', inputBinding: { position: 2, prefix: "--fam" }, doc: "Binary FAM file for plink." }
+  output_basename: { type: 'string?', default: "plink", inputBinding: { position: 2, prefix: "--out"}, doc: "Basename for plink binary files." }
+
+  # GENOME
+  genome:
+    type:
```

Comment:
Hmmm, the "sub-docs" on the record don't seem to show up in the cavatica form...do they using cwltool? Might wanna consider a summary of options at the `genome` doc level

Reply:
That sucks. I'll see what I can do.

Reply:
Added it to the plink subworkflow doc.